server/src/features/features_utils.rs

```diff
@@ -170,31 +170,37 @@ impl FeaturesUtils {
                     }
                     if let Some(model) = session.sync_odoo.models.get(&str).cloned() {
                         let main_classes = model.borrow().get_main_symbols(session, from_module.clone());
-                        for main_class in main_classes.iter().map(|main_class_rc| main_class_rc.borrow()) {
+                        for main_class_rc in main_classes.iter() {
+                            let main_class = main_class_rc.borrow();
                             if let Some(main_class_module) = main_class.find_module() {
                                 value += format!("Model in {}: {}  \n", main_class_module.borrow().name(), main_class.name()).as_str();
                                 if main_class.doc_string().is_some() {
                                     value = value + "  \n***  \n" + main_class.doc_string().as_ref().unwrap();
                                 }
-                                let mut other_imps = model.borrow().all_symbols(session, from_module.clone());
+                                let mut other_imps= model.borrow().all_symbols(session, from_module.clone()).into_iter().filter_map(|(sym, needed_module)| {
+                                    if Rc::ptr_eq(&sym, main_class_rc) {
+                                        None // Skip main_class
+                                    } else {
+                                        Some((sym.borrow().find_module().unwrap().borrow().name().clone(), needed_module))
+                                    }
+                                }).collect::<HashSet<_>>().into_iter().collect::<Vec<_>>();
```

Comment:
I can replace collecting to hashset then to vector just to get unique elements by using [itertools::unique](https://docs.rs/itertools/0.9.0/itertools/trait.Itertools.html#method.unique) .   
How do you feel about adding a new dependency ?

Reply:
Reminder-to-self to use itertools since it is added in a sepearate PR

Reply:
I did not add it in my pr, so you can do it here :)

Reply:
Oki, will do it and ping you here once it is ready for review

Reply:
@fda-odoo Updated